content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -885,9 +885,9 @@ import { Prisma } from '@prisma/client'
 
 prisma.log.findMany({
   where: {
-    data: {
-      meta: Prisma.AnyNull,
-    },
+     meta: {
+       equals: Prisma.AnyNull,
+     },
```

Comment:
Fixing the indentation of the changed code snippet:
```suggestion
    meta: {
      equals: Prisma.AnyNull,
    },
```

Reply:
my bad thought it was already being auto indented with prettier after husky pre-commit.